keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/handlers/LifecycleEventNotificationHandler.kt

```diff
@@ -14,13 +14,13 @@ import org.springframework.stereotype.Component
  * Sends notification for a lifecycle event, like bake / build failures
  */
 @Component
-class LifecycleEventNotificationHandler (
+class LifecycleEventNotificationHandler(
   private val slackService: SlackService,
   @Value("\${spinnaker.baseUrl}") private val spinnakerBaseUrl: String,
   private val gitDataGenerator: GitDataGenerator
 ) : SlackNotificationHandler<SlackLifecycleNotification> {
 
-  override val type: NotificationType = NotificationType.LIFECYCLE_EVENT
+  override val type = listOf(NotificationType.LIFECYCLE_EVENT)
```

Comment:
same nit re `type` to `types`

Reply:
going to stop commenting this LOL because it needs to be changed on the base class